pkg/config/cmd_terminal.go

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright (c) 2019-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/che-incubator/che-workspace-operator/pkg/apis/workspace/v1alpha1"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	// property name for value with default dockerimage component if noone is used with che-incubator/command-line-terminal
+	defaultTerminalDockerimageProperty = "che.workspace.default_dockerimage.che-incubator.command-line-terminal"
+	defaultTerminalDockerimage         = "" +
+		"type: dockerimage\n" +
+		"memoryLimit: 256Mi\n" +
+		"alias: dev\n" +
+		"image: 'registry.redhat.io/codeready-workspaces/plugin-openshift-rhel8:2.1'\n" +
+		"args: ['tail', '-f', '/dev/null']\n" +
+		"env:\n" +
+		"- value: '\\[\\e[34m\\]>\\[\\e[m\\]\\[\\e[33m\\]>\\[\\e[m\\]'\n" +
+		"  name: PS1\n"
```

Comment:
It might be cleaner to use raw strings here:
```go
defaultTerminalDockerimage = 
`type: dockerimage
memoryLimit: 256Mi
alias: dev
image: 'registry.redhat.io/codeready-workspaces/plugin-openshift-rhel8:2.1'
args: ['tail', '-f', '/dev/null']
env:
  - name: PS1
    value: '\\[\\e[34m\\]>\\[\\e[m\\]\\[\\e[33m\\]>\\[\\e[m\\]' 
`
```

Reply:
I think it would be even a safe option to use go object here, like
```
var (
	defaultTerminalDockerimage         = &v1alpha1.ComponentSpec {
		Type: "dockerimage",
		MemoryLimit: "256Mi",
		Alias: "dev",
		Image: "registry.redhat.io/codeready-workspaces/plugin-openshift-rhel8:2.1",
		Args: []string{"tail", "-f", "/dev/null"},
		Env: []v1alpha1.Env{
			{
				Name: "PS1",
				Value: "\\[\\e[34m\\]>\\[\\e[m\\]\\[\\e[33m\\]>\\[\\e[m\\]",
			},
		},
	}
)
```
Do you like it or raw string would be better?

Reply:
Yeah, using a struct is even better here.

Reply:
Yeah I agree with struct as well